### PR TITLE
Feat/password reset rework

### PR DESCRIPTION
**Pull request by joaoqueiroz95**
[password-reset.webm](https://github.com/joel-gouveia/frs-booking/assets/112548481/f5db84d7-14c5-4696-88e5-4da197b34c44)

### CODE REVIEW

**Comment by joaoqueiroz95** [comment]
[Password Reset Screen Rework](https://trello.com/c/i0J9f0TB/15-password-reset-screen-rework)

File: src/screens/Login.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function LoginScreen() {
   return (
     <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
       <VStack flex={1} p={20} style={styles.container}>
-        <VStack alignItems="center" gap={10} mt={50} mb={100}>
+        <VStack alignItems="center" gap={10} mt={50} mb={100} style={styles.header}>
```
Comment (mathvaillant) [inline src/screens/Login.tsx:39]: Remove this style and do 

```
<VStack ... flex={1} ... >
```

**Review by mathvaillant — APPROVED** [review]
Besides the prop flex on the VStack, LGTM!

File: src/screens/Login.tsx (hunk 1 above)
Comment (joaoqueiroz95) [reply src/screens/Login.tsx:39]: Done

**Review by mathvaillant — APPROVED** [review]

**Review by joel-gouveia — APPROVED** [review]
LGTM!

**Pull request merged**